### DISCUSSION TITLE
STAKE BELOW MIN and  INTERNAL SERVER ERROR.

### DISCUSSION THREAD

**Discussion by FireWin93** [cloudbet/docs#63] [General]
Hello,
I would like to understand why when I bet in USDT I receive the error message "STAKE BELOW MIN", an error that I do not receive on an equivalent amount in ETH. Is it possible to set up something to no longer have a minimum stake constraint in USDT?
I am receiving in ETH "INTERNAL SERVER ERROR" error message. What is causing this error? When will the problem be resolved?
Best regards,
FireWin.

**Comment by kgravenreuth** [comment]
Hi @FireWin93 

Apologies for this issue which you might have observed momentarily a few days ago. This issue was already addressed by our engineering team last week and should not be an issue anymore. 

If you do observe any such issues again, please give us specific examples with request/response details thank you.

Regards,
Klaus Gravenreuth
Product & Engineering @ Cloudbet
